orchagent/flexcounterorch.cpp

```diff
@@ -144,6 +149,8 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                     fieldValues.emplace_back(FLEX_COUNTER_STATUS_FIELD, value);
                     m_flexCounterGroupTable->set(flexCounterGroupMap[key], fieldValues);
                 }
+                else if(field == FLEX_COUNTER_DELAY_STATUS_FIELD)
+                {}
```

Comment:
add at least comment why this is ignored?

Reply:
Done.

Reply:
i more was thinking about commend in IF itself where ou have {}, 
```
{
 // comment
}
```

Reply:
how about now?